.codecov.yml

```diff
@@ -5,6 +5,7 @@ coverage:
   ignore:
     - "*_test.go"
     - "./mocks/*"
+    - "./app/internal/easyid"
```

Comment:
because the SDK makes things private the only way to test it would be to mock the server side (which would be very annoying) or write our own implementation